openlibrary/plugins/worksearch/code.py

```diff
@@ -278,7 +293,7 @@ def build_q_list(param):
                     q_list.append("(author_name:(%(name)s) OR author_alternative_name:(%(name)s))" % {'name': v})
 
         check_params = ['title', 'publisher', 'oclc', 'lccn', 'contribtor', 'subject', 'place', 'person', 'time']
-        q_list += ['%s:(%s)' % (k, param[k]) for k in check_params if k in param]
+        q_list += ['%s:(%s)' % (k, depunctuate(param[k])) for k in check_params if k in param]
```

Comment:
```suggestion
        q_list += [
            '%s:(%s)' % (k, depunctuate(param[k])) for k in check_params if k in param
        ]
```
Use psf/black for format long line to placate Travis CI.